driver/src/test/unit/com/mongodb/GridFSUploadStreamSpecification.groovy

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb
+
+import com.mongodb.client.MongoCollection
+import org.bson.BsonDocument
+import org.bson.types.ObjectId
+import spock.lang.Specification
+
+class GridFSUploadStreamSpecification extends Specification {
```

Comment:
Couldn't all these tests assert the actual documents written?  Asserting just that any document is written is not a strong enough test.